lib/rust/ensogl/component/list-view/src/lib.rs

```diff
@@ -480,11 +488,16 @@ where E::Model: Default
             view_info <- all_with(&view_y.value,&frp.size, |y,size|
                 View{position_y:*y,size:*size}
             );
+            default_style_prefix <- init.constant(list_view_style::HERE.str.to_string());
```

Comment:
`list_view_style::HERE.str` - let it be a constant placed in the top of the file.